### PR TITLE
Automated tests on PR

### PR DESCRIPTION
**Pull request by kwinto**
This PR adds a "github workflow" to run our test suite after creating a PR. Subsequent pushes to the branch of PR will retrigger test run.

Also fixes currently failing test (minor issue).

After this get merged, we need to setup branch protection on `master` to prohibit non-admin merge without successful test run.

### CODE REVIEW

File: .github/workflows/cypress.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: Run Cypress
+on: [pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+      - run: npm install
```
Comment (pedily) [inline .github/workflows/cypress.yml:9]: shouldn't this be `npm ci`?

File: .github/workflows/cypress.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+name: Run Cypress
+on: [pull_request]
```
Comment (pedily) [inline .github/workflows/cypress.yml:2]: will this also reset / trigger again if new content is pushed to a pr?

File: .github/workflows/cypress.yml (hunk 1 above)
Comment (kwinto) [reply .github/workflows/cypress.yml]: fixed

File: .github/workflows/cypress.yml (hunk 2 above)
Comment (kwinto) [reply .github/workflows/cypress.yml:2]: see pr description

**Pull request merged**